cmd/jujud-controller/agent/machine/manifolds_test.go

```diff
@@ -1058,32 +1110,39 @@ var expectedMachineManifoldsWithDependenciesIAAS = map[string][]string{
 		"controller-agent-config",
 		"db-accessor",
 		"file-notify-watcher",
-		"provider-tracker",
 		"is-controller-flag",
 		"lease-manager",
 		"object-store-s3-caller",
-		"provider-service-factory",
+		"object-store-services",
 		"query-logger",
 		"s3-http-client",
-		"service-factory",
```

Comment:
We no longer require the full-blown service factory for the object store. This means we can drop the provider tracker as well!